src/pages/styles/HomePage.css

```diff
@@ -1,74 +0,0 @@
-* {
-  font-family: 'HelveticaNeue-Light', 'Helvetica Neue Light', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif;
-}
-
-/* @media (max-width: 1100px) {
-  .welcome-container-border {
-    padding: 1 0.75rem 1 0.75rem !important;
-    width: 80% !important;
-    z-index: -1 !important;
-    height: 5% !important;
-    margin-top: 10rem !important;
-  }
-} */
-
-.homepage-container {
-  /* display: flex; */
-  /* flex-direction: column; */
-  /* align-items: center;
-  justify-content: center; */
-}
-
-.welcome-container-border {
-  background-color: var(--gl-light-gray);
-  padding: 1rem;
-  width: 1100px;
-  height: 90%;
-  margin: auto;
-
-  border-radius: 17.5px;
-  bottom: 20%;
-  margin-top: 3rem;
-}
-
-.welcome-container {
-  width: auto;
-  height: 100%;
-  padding: 1rem 2rem 1rem 2rem;
-  text-align: center;
-  border-radius: 10px;
-  background-color: white;
-
-  align-items: center;
-  justify-content: center;
-  flex-direction: column;
-  top: 50%;
-}
-
-.homepage-title {
-  font-size: 55px;
-  margin-bottom: 0;
-}
-.homepage-subheading {
-  font-size: 30px;
-}
-
-@media (max-width: 1100px) {
-  .welcome-container-border {
-    width: 550px;
-  }
-}
-@media (max-width: 550px) {
-  .welcome-container-border {
-    width: 300px;
-  }
-}
-
-@media screen and (max-width: 800px) {
-  .homepage-title {
-    font-size: 40px;
-  }
-  .homepage-subheading {
-    font-size: 20px;
-  }
-}
```

Comment:
Removed as the remaining CSS was just for the hard coded TextIntro that has now be moved to TextIntro.css.